app/src/ui/history/commit-summary.tsx

```diff
@@ -179,6 +183,41 @@ export class CommitSummary extends React.Component<ICommitSummaryProps, ICommitS
     )
   }
 
+  private onShowCommitOptions = (event: React.MouseEvent<any>) => {
+    event.preventDefault()
+
+    const items: IMenuItem[] = [
+      {
+        label: __DARWIN__ ? 'Revert this Commit' : 'Revert this commit',
+        action: this.onRevertCommit,
+      },
+      { type: 'separator' },
+      {
+        label: 'Copy SHA',
+        action: this.onCopySHA,
+      },
+      {
+        label: 'View on GitHub',
+        action: this.onViewOnGitHub,
+        enabled: !this.props.isLocal || !this.props.repository.gitHubRepository,
```

Comment:
Shouldn't this be an `&& this.props.repository.gitHubRepository`? As written, wouldn't it be enabled if the commit isn't local but the repository doesn't have a `gitHubRepository`?